backend/LexCore/ServiceInterfaces/IPermissionService.cs

```diff
@@ -5,16 +5,19 @@ namespace LexCore.ServiceInterfaces;
 public interface IPermissionService
 {
     ValueTask<bool> CanSyncProject(string projectCode);
+    // CanSyncProject(Guid) does NOT check permissions for org managers, because that requires async DB access
     bool CanSyncProject(Guid projectId);
+    // CanSyncProjectAsync(Guid) does all the checks from CanSyncProject, plus allows org managers to access the org as well
```

Comment:
make this proper C# documentation so it shows up when calling this method

Reply:
Done in commit 08bc5f87.